src/services/theme/hooks.tsx

```diff
@@ -19,11 +19,13 @@ import {
   EuiThemeComputed,
 } from './types';
 
-export const useEuiTheme = <T extends {} = {}>(): {
+export interface UseEuiTheme<T extends {} = {}> {
```

Comment:
Does it make sense to append `Props`?

```suggestion
export interface UseEuiThemeProps<T extends {} = {}> {
```

Reply:
I also struggled with this. I don't think `props` makes sense because this is the return value not the input options.

Reply:
Ah right, I don't know what the pattern is for that then